libs/extensions/include/NanoDimuonVertex.hpp

```diff
@@ -15,31 +15,58 @@ typedef Collection<std::shared_ptr<NanoDimuonVertex>> NanoMuonVertices;
 
 class NanoDimuonVertex {
  public:
-  NanoDimuonVertex(std::shared_ptr<PhysicsObject> physicsObject_);
+  NanoDimuonVertex(std::shared_ptr<PhysicsObject> physicsObject_, const std::shared_ptr<Event> event);
```

Comment:
Something I didn't catch before: what is the `physicsObject_` here? Could it have a more meaningful name?

Reply:
It's the muonvertex collection, I'm just following the same logic as for the other nano-classes, like NanoMuon(std::shared_ptr<PhysicsObject> physicsObject_);